tasks/pipeline-osdeps-MCPClient.yml

```diff
@@ -27,6 +27,13 @@
       when:
         - "MCPClient.osdeps_packages"
 
+    - name: "Install MCPClient ext. package deb file deps."
+      apt:
+        deb: "{{ item }}"
+      with_items: "{{ MCPClient.osdeps_debfiles }}"
+      when:
+        - "MCPClient.osdeps_debfiles"
+
```

Comment:
Yep, you're right. I had to add an `is defined` check. That made provisioning work even when no osdeps_debfiles attribute exists in the Ubuntu osdeps object.